rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/util/UnityUtils.kt

```diff
@@ -43,6 +43,21 @@ fun MutableList<String>.withProjectPath(project: Project) : MutableList<String>
     return this
 }
 
+fun MutableList<String>.withBatchMode(project: Project) : MutableList<String> {
+    this.add("-batchmode")
+    return this
+}
+
+fun MutableList<String>.withRunTests(project: Project) : MutableList<String> {
+    this.add("-runTests")
+    return this
+}
+
+fun MutableList<String>.withTestResults(project: Project) : MutableList<String> {
+    this.addAll(listOf("-testResults", project.solutionDirectory.resolve("results.xml").canonicalPath))
```

Comment:
I am not sure that results.xml in repo root is a good place, what's about creating seperated folder. e.g UnitTestsResults? Or put results in library

Reply:
I have changed it to point to `Temp` folder in the solution. That Temp folder is created anyway by Unity.